features/max_login_attempts.feature

```diff
@@ -0,0 +1,44 @@
+Feature:
+  In order to protect the APP
+  I should be blocked if I try to login too many time
+
+  Scenario: I'm block after 5 attempts
+    Given the following fixtures are loaded:
+      | LoadAdherentData |
+    Given I am on "/espace-adherent/connexion"
+
+    When I fill in the following:
+      | _adherent_email    | luciole1989@spambox.fr |
+      | _adherent_password | wrongPassword          |
+    And I press "Je me connecte"
+    Then I should see "Oups! Vos identifiants sont invalides."
+
+    When I fill in the following:
+      | _adherent_email    | luciole1989@spambox.fr |
+      | _adherent_password | wrongPassword          |
+    And I press "Je me connecte"
+    Then I should see "Oups! Vos identifiants sont invalides."
+
+    When I fill in the following:
+      | _adherent_email    | luciole1989@spambox.fr |
+      | _adherent_password | wrongPassword          |
+    And I press "Je me connecte"
+    Then I should see "Oups! Vos identifiants sont invalides."
+
+    When I fill in the following:
+      | _adherent_email    | luciole1989@spambox.fr |
+      | _adherent_password | wrongPassword          |
+    And I press "Je me connecte"
+    Then I should see "Oups! Vos identifiants sont invalides."
+
+    When I fill in the following:
+      | _adherent_email    | luciole1989@spambox.fr |
+      | _adherent_password | wrongPassword          |
+    And I press "Je me connecte"
+    Then I should see "Oups! Vos identifiants sont invalides."
+
+    When I fill in the following:
+      | _adherent_email    | luciole1989@spambox.fr |
+      | _adherent_password | EnMarche2017           |
+    And I press "Je me connecte"
+    Then I should see "Oups! Vos identifiants sont invalides."
```

Comment:
The error message should be different no ?

Reply:
No we don't tell the user he reach the max limit

Reply:
Not sure it's a good idea, it can be really confusing for the end user. Specially with a limit of 5 attempts